### PR TITLE
KOGITO-2833: Remove Enzyme as snapshot serializer

### PR DESCRIPTION
**Pull request by tiagobento**
Enzyme snapshots contained information such as stringified JSON, but we're actually only interested in the produced DOM hierarchy. Also, this states `@testing-library/react` as our default tool for testing components unitarily.

### CODE REVIEW

**Comment by tiagobento** [comment]
cc @caponetto @manstis @paulovmr @ederign

**Review by ljmotta — CHANGES_REQUESTED** [review]
Nice PR @tiagobento!

File: packages/microeditor-envelope/jest/jest-env-setup.js [hunk 1]
```diff
@@ -13,8 +13,3 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
```
Comment (ljmotta) [inline packages/microeditor-envelope/jest/jest-env-setup.js:16]: The file is empty, it shouldn't be removed?

File: packages/microeditor-envelope/src/__tests__/EditorEnvelopeController.test.tsx [hunk 2]
```diff
@@ -16,100 +16,67 @@
...
-    new ResourceContentServiceCoordinator(),
-    new DefaultKeyboardShortcutsService({
-      editorContext: { channel: ChannelType.VSCODE, operatingSystem: OperatingSystem.WINDOWS }
-    })
-  );
-});
+describe("EditorEnvelopeController", () => {
+  beforeEach(() => {
+    loadingScreenContainer = document.body.appendChild(document.createElement("div"));
+    loadingScreenContainer.setAttribute("id", "loading-screen");
```
Comment (ljmotta) [inline packages/microeditor-envelope/src/__tests__/EditorEnvelopeController.test.tsx:38]: Is this necessary? The LoadingScreen is rendered on the EditorEnvelopeController.

**Review by ljmotta — CHANGES_REQUESTED** [review]
I've made some comments inline

File: packages/microeditor-envelope/src/__tests__/EditorEnvelopeController.test.tsx (hunk 2 above)
Comment (tiagobento) [reply packages/microeditor-envelope/src/__tests__/EditorEnvelopeController.test.tsx:38]: Yes. This id is used by `SpecialDomElements.ts` to locate a container for the React Portal we create to render the LoadingScreen component.

File: packages/microeditor-envelope/jest/jest-env-setup.js (hunk 1 above)
Comment (tiagobento) [reply packages/microeditor-envelope/jest/jest-env-setup.js:16]: Good catch. Removed.

**Review by ljmotta — APPROVED** [review]

**Review by ederign — APPROVED** [review]

**Pull request merged**